dts/msm8226/msm8926-v1-mtp.dts

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	compatible = "qcom,msm8926-mtp", "qcom,msm8926";
+	qcom,msm-id = <200 0>;
+	qcom,board-id = <8 0>;
+
+	// All the Nokia devices are expected to use lk1st, which passes the compatible to lk2nd, so no need for extra device matching.
```

Comment:
(re-adding it here since GH works now)
I'd prefer the comments/code to be folded at 80 where possible. Also could edit the readme :)